src/frontend/src/storage/index.test.ts

```diff
@@ -196,3 +288,25 @@ function readLocalStorage(): LocalStorage {
 
   return ls;
 }
+
+/** Indexed DB */
+
+type IndexedDB = Record<string, unknown>;
+
+const setIndexedDB = async (db: IndexedDB) => {
```

Comment:
why not `setMany` and `getMany` for set|getIndexedDB, wouldn't it be more accurate given that it's atomic?

Reply:
This felt like the easier to write & read but I don't feel strongly about it

Reply:
No strong opinion neither given it's a test